participants/ipfsMart.md

```diff
@@ -0,0 +1,49 @@
+
+(2) Link to Project UI.  
+http://ai.ipfsmart.io/filecoin_index
+
+
+(3) What does your application/UI do?   
+**ipfsMart** provides all essentials for users to develop AI applications as a routine AI platform, and it further provides the following activities.  
+First, a data scientist can make an application development task as an open project, so anyone can contribute to this task by providing data and making annotations.  
+Second, AI models could be trained based on the whole set or a subset of the provided data. Any individual's any contribution (i.e., data provision and annotation) would be linked to the corresponding model and be quantified as a portion of options.  
+Third, because the training process continuously provides intermediate models, **ipfsMart** takes advantage of the sufficient storage space of IPFS to store all all intermediate models for potential use instead of dropping them off as the routine operation.  
+Fourth, once a final model/application is obtained after sufficient training and verification, **ipfsMart** deploys it onto different IPFS nodes rather than deploy it onto a single machine as the routine operation. As a result, when multiple customers access to the AI model/application synchronously, **ipfsMart** can allocate the models of different nodes to deal with different customer's access requirements.  
+Finally, each access to the model could generate a certain quantity of profit, and the profit would be given to all contributors (i.e., data scientist, dat provider and data annotator) according to the potions they hold.  
+ 
+(4) If your project is using a curated dataset, which one are you using?   
+We do not use curated datasets.
+
+(5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.   
+We only provide a platform for carrying out the AI projects. We do not own the data, and the data rights belong to our users.    
+
+(6) How much data are you planning to store to the Filecoin network during the Slingshot competition?   
+We wish to contribute to 0.5-1T quantity of data.   
+
+(7) How are you structuring the data?   
+Most of these data is unstructured, like videos and images.   
+
+(8) What pre-processing are you doing before ingesting the data?   
+We transfer user's data, intermediate models and final models/applications to the IPFS network and assign a unique identifier for each piece of data.   
+
+(9) What tech stack will you use to store the data?   
+lotus
```

Comment:
Just confirming that you'll be deploying on the Filecoin network (a lot of the description above was about IPFS).  Regardless IPFS may be a great caching layer for hot retreival.